src/__integrations__/pagopaApi.server.integration.test.ts

```diff
@@ -119,4 +120,118 @@ describe('Payment Manager Client', () => {
     expect(responseP.map((myRes: any) => myRes.status).getOrElse(404)).toEqual(200);
     await stubServerTerminator.terminate();
   });
+
+  it('should return a left either (error 422) when start-session is called with a void payload', async () => {
```

Comment:
Really interesting test 💪 however the description does not represent it, for example from the description it seems that the test is focused on checking the response status `422` but `500` is also checked.

Reply:
@infantesimone In order to have more descriptive names we have to choose a strategy to better organize tests. In my opinion we have two options:
- A different test file for each endpoint,
- A different describe block inside `pagopaApi.server....` for each endpoint.
What's the best option in your opinion?

Reply:
Good point, to avoid a large file I think it is better to divide into several files because there will be at least 6 other PM endpoints to test. So, we could create something like this:

- pagopaApi.startSession.integration.test.ts
- pagopaApi.checkPayment.integration.test.ts
- pagopaApi.acceptTerms.integration.test.ts
- .....

Reply:
@infantesimone @pasqualespica  Ok, I'll refactor as you suggest